docs/README.md

```diff
@@ -421,6 +421,8 @@ If you are running the container in a virtualized environment (Hyper-V, VirtualB
 
 [acme-companion](https://github.com/nginx-proxy/acme-companion) is a lightweight companion container for the nginx-proxy. It allows the automated creation/renewal of SSL certificates using the ACME protocol.
 
+By default nginx-proxy generates location blocks to handle ACME HTTP Challenge. To disable this behavior set `ACME_HTTP_CHALLENGE=false` (default `true`) at the nginx-proxy container level or at the app container one.
+
```

Comment:
Either we choose to not document this feature for now because it won't be complete, or we alter the text, for example like this:
```suggestion
By default nginx-proxy generates location blocks to avoid redirecting ACME HTTP challenge to HTTPS when using `HTTPS_METHOD=redirect` (the default value). To additionally generate location blocks for HTTPS, set `ACME_HTTP_CHALLENGE=true` (default `false`) at the nginx-proxy container level or at the app container one.
```

Reply:
There are broken use cases with both defaults:

**true**
acme_companion configures an extraneous HTTP Challenge location block which is not valid.

**false**
Use cases where certificates are not generated with acme_companion loose HTTP Challenge configuration and may fail to renew their certificates.

Why not fixing acme_companion instead?